### PR TITLE
Changes the material costs of the nuclear charge

### PR DESCRIPTION
**Pull request by DimWhat**
[Balance]
## About the PR 
Alters the material costs of a nuclear charge from 80 generic materials to a more complex recipe which is:
27 Extreme Power Source
25 Dense Metal
13 High Energy Conductor
15 Extremely Dense Crystalline Matter

## Why's this needed? 
A nuclear charge is about the size of a canbomb. You shouldn't be able to fabricate something like that without some effort in my opinion.

## Changelog

```changelog
(u)DimWhat
(*) Made fabricating a nuclear charge take more difficult materials
```

### CODE REVIEW

**Comment by penny64** [comment]
Even for a nuke this recipe is totally insane

File: code/modules/networks/computer3/mainframe2/misc_terms.dm [hunk 1]
```diff
@@ -1037,7 +1037,7 @@
 
 #define DISARM_CUTOFF 10 //Can't disarm past this point! OH NO!
 
-	mats = 80 //haha this is a bad idea
+	mats = list("POW-3"=113, "MET-3" = 134, "CON-2" = 117, "DEN-3" = 142) //haha this is a bad idea
```
Comment (penny64) [inline code/modules/networks/computer3/mainframe2/misc_terms.dm:1040]: POW-3 and MET-3, that's basically just all cerenkite in this recipe

**Review by penny64 — CHANGES_REQUESTED** [review]

File: code/modules/networks/computer3/mainframe2/misc_terms.dm (hunk 1 above)
Comment (DimWhat) [reply code/modules/networks/computer3/mainframe2/misc_terms.dm:1040]: Cerenkite does not meet the requirements of tier 3 metal. Though it meeting tier 3 power is something I didn't know.

**Comment by unfunnyperson** [comment]
the change feels a little too small for a canbomb sized blast, I suggest bumping it to around 200 total mats instead

**Comment by COOLVAPE** [comment]
80 mats, some of which are rare seems pretty reasonable for something that's already quite rare to produce - the rare requirements make it so you can't roundstart produce the nuke, which seems reasonable.

**Comment by Crackaduck** [comment]
Price is just too much for me to be comfortable with, especially the EDCM, which is already hard to get as is.

**Comment by Tarmunora** [comment]
now that syndicate device analyzers can't be reliably acquired by non-traitor-mechanics, 80 mats is pretty reasonable

**Pull request merged**